pdns/recursordist/rec-tcpout.cc

```diff
@@ -51,33 +51,33 @@ void TCPOutConnectionManager::cleanup(const struct timeval& now)
   }
 }
 
-void TCPOutConnectionManager::store(const struct timeval& now, const ComboAddress& ip, Connection&& connection)
+void TCPOutConnectionManager::store(const struct timeval& now, const pair_t& pair, Connection&& connection)
```

Comment:
maybe a comment explaining what pair is, or a small (free) typedef to give both ends of the pair a readable name?